class.two-factor-core.php

```diff
@@ -230,10 +230,6 @@ public static function wp_login( $user_login, $user ) {
 	 * @param WP_User $user WP_User object of the logged-in user.
 	 */
 	public static function show_two_factor_login( $user ) {
-		if ( ! function_exists( 'login_header' ) ) {
-			require_once( ABSPATH . WPINC . '/functions.wp-login.php' );
```

Comment:
Incidentally, when I first wrote this, I don't know why I thought `wp-includes/functions.wp-login.php` existed -- it never has best as I can tell.